Readme.md

```diff
@@ -1,24 +1,20 @@
-# WebMemex/memextension
+# WorldBrain's WebMemex
```

Comment:
I interpreted it as "WorldBrain's version of the WebMemex", but I see that WorldBrain's WebMemex sounds too much as if it is our work completely.